docs/source/project_documentation/user_guide/model_definitions.md

```diff
@@ -60,8 +60,20 @@ triggers (`when`), and post-conditions (`then`) for the behavior.
     :lines: 6-38
     :emphasize-lines: 19, 21, 23, 25, 28, 30, 32
 ```
+### External Models
+
+An *External Model* is a `model` that does not include an *Acceptance Scenario*.  These types of `model`s are used to create simplified interfaces representing external components or actors.  A typical use case for External Model is to represent a user or external messenger that is not managed by AaC.  It allows you to quickly create `model`s without having to fully implement them.
```

Comment:
Reviewing this in more than a quick skim like I gave it yesterday.

```suggestion
An *External Model* is a `model` that does not include an *Acceptance Scenario*.  These types of models are used to create simplified interfaces representing external components or actors.  A typical use case for an External Model is to represent a user or external messenger that is not managed by AaC.  It allows you to quickly create models without having to fully implement them.
```

Some of the model references made were not about the `model` field, but a model representing a system or interface to a system.